plugins/obs-filters/data/blend_mul_filter.effect

```diff
@@ -3,6 +3,7 @@ uniform texture2d image;
 
 uniform texture2d target;
 uniform float4 color;
+uniform int opacity;
```

Comment:
opacity should be a float.

Reply:
The creator of this plugin created opacity as an integer in the GUI.  Would that enhancement require another PR?

Reply:
As it is part of the bug fix (since opacity is actually used now), it would not.

Reply:
Prior to pushing this PR up, I had tried a float, but it made no difference. 

It even made it more troublesome and time consuming. Because, it seems that 15.15% might be better than 15%... but you can't tell the difference so you strain your eyes trying to see an improvement.

If changed, the slider would still go by 1.0 step values to prevent insanity.

Reply:
slider set to step at 0.05... I think everyone will be happy now!